tests/Unit/inc/classes/CDN/RocketCDN/APIClient/getPricingData.php

```diff
@@ -3,11 +3,11 @@
 namespace WP_Rocket\Tests\Unit\inc\classes\CDN\RocketCDN\APIClient;
 
 use WPMedia\PHPUnit\Unit\TestCase;
-use WP_Rocket\CDN\RocketCDN\APIClient;
+use WP_Rocket\Engine\CDN\RocketCDN\APIClient;
```

Comment:
Move this test and all unit tests for RocketCDN into `tests/Unit/inc/CDN/RocketCDN/...`.